api/src/cohort/cohortValidatorTemp.ts

```diff
@@ -0,0 +1,7 @@
+// Function will be updated in the next PR, for the time being it's useful to test typing import from snu-lib
+
+import { CohortDto } from "snu-lib/src/dto";
```

Comment:
```suggestion
import { CohortDto } from "snu-lib";
```
🤔 on ne devrait pas avoir de `src` ou de sous-repertoire dans l'import

Reply:
C'est un type typescript qui, pas définition n'est pas dans le common-js, d'où le chemin nécessaire pour importer ce type. On peut regarder ensemble si tu veux.

Reply:
Yop, vous vous êtes décidé la dessus ?
On peut pas virer rollup et le commonjs du coup avec cette PR ?